service/src/test/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/UploadAdditionalApplicationsServiceTest.java

```diff
@@ -55,7 +56,8 @@
 @ContextConfiguration(classes = {
     UploadAdditionalApplicationsService.class,
     FixedTimeConfiguration.class,
-    DocumentUploadHelper.class
+    DocumentUploadHelper.class,
+    ObjectMapper.class
```

Comment:
is there a reason this is using the spring extension and not just using the mockito extension and injecting the mocks

Reply:
not sure why spring extension was used. I'll create a separate PR to use MockitoExtension as we are not sure if this PR will be merged